ui-lib/src/main/java/co/electriccoin/zcash/ui/common/model/WalletSnapshot.kt

```diff
@@ -42,6 +42,8 @@ fun WalletSnapshot.canSpend(amount: Zatoshi): Boolean = spendableBalance() >= am
 
 fun WalletSnapshot.totalBalance() = orchardBalance.total + saplingBalance.total + transparentBalance
 
+fun WalletSnapshot.totalSpendableBalance() = orchardBalance.total + saplingBalance.total
```

Comment:
I'd not call it spendable, as it's composed of just total balances, not available ones. Let's call it `totalShieldedBalance`?

Reply:
✅